packages/x-data-grid/src/models/colDef/gridColDef.ts

```diff
@@ -355,6 +355,16 @@ export type GridColDef<R extends GridValidRowModel = any, V = any, F = V> =
   | GridActionsColDef<R, V, F>
   | GridSingleSelectColDef<R, V, F>;
 
+/**
+ * Column Definition interface used for the list view column.
+ * @demos
+ *   - [List view](/x/react-data-grid/list-view/)
+ */
+export type GridListColDef<R extends GridValidRowModel = any, V = any, F = V> = Pick<
+  GridBaseColDef<R, V, F>,
+  'field' | 'renderCell' | 'align' | 'cellClassName' | 'display'
```

Comment:
is there a reason why these are being picked up.
in the example only `field` and `renderCell` are being used.
is there plan to support this in a more out-of-box way where you might not need custom renderer for simple use cases?